exercises/practice/two-bucket/two_bucket_types.bal

```diff
@@ -0,0 +1,7 @@
+public type TwoBucketSolution record {|
+    int moves;
+    string goalBucket;
+    int otherBucket;
+|};
+
+public type TwoBucketResult TwoBucketSolution|error;
```

Comment:
Creating a union type consisting of a data type and an error is not the best practice in Ballerina. The better way is to use the union type directly in the function return type.